File: app/src/main/java/com/soft/credit911/Login/Adpater/ViewPagerAdapter.java
package com.soft.credit911.Login.Adpater;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.viewpager.widget.PagerAdapter;

import com.soft.credit911.R;

public class ViewPagerAdapter extends PagerAdapter {
    private Context mContext;

    public ViewPagerAdapter(Context context) {
        mContext = context;
    }
    @Override
    public Object instantiateItem(ViewGroup collection, int position) {
        int resId = 0;
        switch (position) {
            case 0:
                resId = R.layout.viewpager_login_scrren_1;
                break;
            case 1:
                resId = R.layout.viewpager_login_screen_2;

                break;
            case 2:
                resId = R.layout.viewpager_login_screen_3;
                break;
        }
        LayoutInflater inflater = LayoutInflater.from(mContext);
        ViewGroup layout = (ViewGroup) inflater.inflate(resId, collection, false);
        collection.addView(layout);
        return layout;
    }

    @Override
    public void destroyItem(ViewGroup collection, int position, Object view) {
        collection.removeView((View) view);
    }

    @Override
    public int getCount() {
        return 3;
    }

    @Override
    public boolean isViewFromObject(View view, Object object) {
        return view == object;
    }
}

File: app/src/main/java/com/soft/credit911/adaptor/ViewPagerAdapter.kt
package com.soft.credit911.adaptor

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.viewpager.widget.PagerAdapter
import com.soft.credit911.R

class ViewPagerAdapter(private val mContext: Context) : PagerAdapter() {
    override fun instantiateItem(collection: ViewGroup, position: Int): Any {
        var resId = 0
        when (position) {
            0 -> resId = R.layout.viewpager_login_scrren_1
            1 -> resId = R.layout.viewpager_login_screen_2
            2 -> resId = R.layout.viewpager_login_screen_3
        }
        val inflater = LayoutInflater.from(mContext)
        val layout = inflater.inflate(resId, collection, false) as ViewGroup
        collection.addView(layout)
        return layout
    }

    override fun destroyItem(collection: ViewGroup, position: Int, view: Any) {
        collection.removeView(view as View)
    }

    override fun getCount(): Int {
        return 3
    }

    override fun isViewFromObject(view: View, `object`: Any): Boolean {
        return view === `object`
    }
}